code/src/vs/platform/request/node/proxy.ts

```diff
@@ -18,14 +18,39 @@ function getSystemProxyURI(requestURL: Url, env: typeof process.env): string | n
 	return null;
 }
 
+/** Respects the NO_PROXY environment variable. */
```

Comment:
![image](https://github.com/che-incubator/che-code/assets/5676062/e2a6d4a7-2f63-4829-b304-b6c41844e38f)

I would like to test the changes, but , unfortunately, I have no experience related to a proxied environment.
Could you share more details how it should be tested?

Thanks in advance!